guides/02_Content_Share.md

```diff
@@ -0,0 +1,115 @@
+media files in a meeting. This guide assumes you have already created a meeting and
+added attendees to the meeting (see
+[Getting Started](https://aws.github.io/amazon-chime-sdk-js/modules/gettingstarted.html)
+for more information).
+
+Content share methods are accessed from the
+[audio-video facade](https://aws.github.io/amazon-chime-sdk-js/interfaces/audiovideofacade.html)
+belonging to the meeting session.
+
+#### Share content
+
+Using the audio-video facade, start sharing content by calling
+[startContentShare](https://aws.github.io/amazon-chime-sdk-js/interfaces/audiovideofacade.html#startcontentshare)
+and provide a
+[MediaStream](https://developer.mozilla.org/en-US/docs/Web/API/MediaStream):
+
+```javascript
+const meetingSession = // reference to MeetingSession
+await meetingSession.audioVideo.startContentShare(mediaStream);
+```
+
+To start a screen capture, call the convenience method
+[startContentShareFromScreenCapture](https://aws.github.io/amazon-chime-sdk-js/interfaces/audiovideofacade.html#startcontentsharefromscreencapture).
+
+When calling from a browser, leave `sourceId` empty. This will launch the
+browser's native screen capture picker.
+
+```javascript
+const meetingSession = // reference to MeetingSession
+await meetingSession.audioVideo.startContentShareFromScreenCapture();
+```
+
+When calling from Electron, build a screen capture picker into your application and
+pass the `sourceId` of the chosen screen capture to the method. See the
+[desktop-capture](https://github.com/hokein/electron-sample-apps/tree/master/desktop-capture)
+sample application for more information.
+
+```javascript
+const meetingSession = // reference to MeetingSession
+const sourceId = // get this from your custom Electron screen capture picker
+await meetingSession. audioVideo.startContentShareFromScreenCapture(sourceId);
+```
+
+#### View the content share
+
+Content shares are treated as regular audio-video attendees. The attendee ID of a
+content share has a suffix of
+[#content](https://aws.github.io/amazon-chime-sdk-js/enums/contentshareconstants.html#modality).
+You receive real-time attendee presence and volume indicator callbacks
+for content audio and video tile updates for content video.
+
+To view the content share:
+
+- Create an instance of [AudioVideoObserver](https://aws.github.io/amazon-chime-sdk-js/interfaces/audiovideoobserver.html) that implements [videoTileDidUpdate](https://aws.github.io/amazon-chime-sdk-js/interfaces/audiovideoobserver.html#videotiledidupdate) to receive callbacks when the video tile is created and updated
+- Add the observer with [addObserver](https://aws.github.io/amazon-chime-sdk-js/interfaces/audiovideofacade.html#addobserver) method to receive events.
+- In `videoTileDidUpdate`, bind the video tile to a video element in your app:
+
+```javascript
+const meetingSession = // reference to MeetingSession
+const tileState = //
```

Comment:
tileState is the parameter in videoTileDidUpdate. So should we mention it differently?